source/linux/permissions/permissions.rst

```diff
@@ -0,0 +1,16 @@
+Permissions
+***********
+
+.. warning::
+
+    Linux permissions is a very complex topic. This cheatsheet only covers the basics
+
+Overview
+========
+
```

Comment:
We should also describe what groups are and how to edit them. 
Also root / superuser concept should be described.